include/Model.hpp

```diff
@@ -43,6 +43,15 @@ class Model : public ::Model {
         Load(mesh);
     }
 
+    /*
+     * Load a model from a mesh.
+     *
+     * @throws raylib::RaylibException Throws if failed to load the Modal.
```

Comment:
``` suggestion
     * Overloaded constructor to catch when passing in a `raylib::Mesh`. It expects a `raylib::MeshUnmanaged or `::Mesh`.
     *
     * @throws raylib::RaylibException Since the model takes ownership of the Mesh, a `::Mesh` or `raylib::UnmanagedMesh` is required here.
```